src/main/java/hudson/plugins/git/GitChangeLogParser.java

```diff
@@ -79,8 +83,8 @@ public List<GitChangeSet> parse(@NonNull List<String> changelog) {
     @Override public GitChangeSetList parse(Run build, RepositoryBrowser<?> browser, File changelogFile)
         throws IOException, SAXException {
         // Parse the log file into GitChangeSet items - each one is a commit
-        try (LineIterator lineIterator = FileUtils.lineIterator(changelogFile, "UTF-8")) {
-            return new GitChangeSetList(build, browser, parse(lineIterator));
+        try (Stream<String> lineStream = Files.lines(changelogFile.toPath(), StandardCharsets.UTF_8)) {
+            return new GitChangeSetList(build, browser, parse(lineStream.iterator()));
```

Comment:
Avoids the use of Commons I/O when the standard Java Platform functionality would suffice.